staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -188,8 +188,60 @@ type VirtualMachineInstanceStatus struct {
 	// ActivePods is a mapping of pod UID to node name.
 	// It is possible for multiple pods to be running for a single VMI during migration.
 	ActivePods map[types.UID]string `json:"activePods,omitempty"`
+
+	// VolumeStatus contains the statuses of all the volumes
+	// +optional
+	// +listType=atomic
+	VolumeStatus []VolumeStatus `json:"volumeStatus,omitempty"`
+}
+
+// VolumeStatus represents information about the status of volumes attached to the VirtualMachineInstance.
+// +k8s:openapi-gen=true
+type VolumeStatus struct {
+	// Name is the name of the volume
+	Name string `json:"name"`
+	// Target is the target name used when adding the volume to the VM, eg: vda
+	Target string `json:"target"`
+	// Phase is the phase
```

Comment:
Do you think it makes sense to put a comment here. something like, "Phase is not supported by all volume types", just so there's not an expectation that phase should be set?